### PR TITLE
Fixes #488: Add logging to the driver config (4.1)

### LINKED ISSUE #488: Change the log level of Neo4j driver
This issue is related to Zendesk ticket #14201: https://neotechnology.zendesk.com/agent/tickets/14201

A customer is asking the following:

> How we can change the loglevel of streams.kafka.connect.sink.Neo4jSinkConnector?
> org.neo4j.driver.internal.logging.JULogger info
> I’m seeing  huge number of java tracebacks from transient errors (INFO log level) messages or warnings

## Feature description (Mandatory)

In order to reduce the amount of INFO logs from Neo4j driver, we could change the log level into Neo4jService.kt, via configBuilder, as explained [here](https://community.neo4j.com/t/how-to-disable-log-info-level-message-for-neo4j-java-driver/14738)

Might we think about making this setting configurable?

### PR DESCRIPTION
**Pull request by igorvoltaic**
## Fixes #488

Suggestion to change the log level of Neo4j driver

## Proposed Changes

A brief list of proposed changes in order to fix the issue:

- add log level to configBuilder in Neo4jConnectorConfig.kt
- add option to configure driver log level from connector props
- add incorrect input test

### CODE REVIEW

**Comment by igorvoltaic** [comment]
Hi @conker84! Should I close #494?

**Comment by conker84** [comment]
@igorvoltaic yes please.
One more thing, did you sign the [CLA](https://neo4j.com/developer/cla/)?

**Comment by igorvoltaic** [comment]
> @igorvoltaic yes please. One more thing, did you sign the [CLA](https://neo4j.com/developer/cla/)?

Yes, sent the message

File: kafka-connect-neo4j/src/main/kotlin/streams/kafka/connect/common/Neo4jConnectorConfig.kt [hunk 1]
```diff
@@ -33,6 +35,19 @@ object ConfigGroup {
...
+	INFO(Level.INFO),
+	WARN(Level.WARNING),
+	DEBUG(Level.FINE),
+	TRACE(Level.FINEST);
+
+  // Return 'INFO' if user supplied incorrect level name
+  companion object {
+    fun from(type: String?): DriverLogLevel = values().find { it.name == type } ?: INFO
+  }
+}
```
[the remark this answers is not in the document]
Comment (igorvoltaic) [reply kafka-connect-neo4j/src/main/kotlin/streams/kafka/connect/common/Neo4jConnectorConfig.kt]: @conker84 done

**Comment by igorvoltaic** [comment]
Hey @conker84!
Is this still valid or there are no plans to merge it?

**Pull request closed without merging**